package.json

```diff
@@ -122,6 +122,7 @@
     "compare-versions": "^3.6.0",
     "dompurify": "^2.0.12",
     "element-closest": "^3.0.2",
+    "eslint-plugin-react-hooks": "^4.2.0",
```

Comment:
@danielgent please, add it as a dev dependency.

Reply:
Ooops @eugene-manuilov ! nice spot! :facepalm: . Fixed :heavy_check_mark: